sprint 7/tema 1.js

```diff
@@ -0,0 +1,11 @@
+function divNumber(number){
+
+    if(number % 10 === 0){
+        console.log(true);
```

Comment:
```suggestion
        return true;
```

si mai jos return false